docs/tfengine/schemas/devops.md

```diff
@@ -15,7 +15,7 @@
 | admins_group.owners | Owners of the group. | array(string) | false | - | - |
 | billing_account | ID of billing account to attach to this project. | string | false | - | - |
 | enable_gcs_backend | Whether to enable GCS backend for the devops module.        Defaults to false.<br><br>Since the devops module creates the state bucket, it cannot back up        the state to the GCS bucket on the first module. Thus, this field        should be set to false initially.<br><br>After the devops module has been applied once and the state bucket        exists, the user should set this to true and regenerate the configs.<br><br>To migrate the state from local to GCS, run `terraform init` on the        module. | boolean | false | false | - |
-| parent_id | ID of parent GCP resource to apply the policy.        Can be one of the organization ID or folder ID according to parent_type. | string | false | - | ^[0-9]{8,25}$ |
+| parent_id | ID of parent GCP resource to apply the policy.        Can be one of the organization ID or folder ID according to parent_type.        See <https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy> to learn more about resource hierarchy. | string | false | - | ^[0-9]{8,25}$ |
```

Comment:
just wondering is it possible to remove the extra spaces between the sentences? not a big deal as it doesn't show the extra spaces when rendering in markdown

Reply:
Sure, let me see if I can do it easily and I'll open another PR to master (doing it here would be a lot of overhead for all the files). I agree it's not a big deal since it doesn't really create a problem.

Reply:
Solved by in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/969